Src/Support/Google.Apis.Auth/OAuth2/OIdCToken.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+Copyright 2020 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Util;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Apis.Auth.OAuth2
+{
+    /// <summary>
+    /// Represents an OIdCToken.
+    /// </summary>
+    public sealed class OIdCToken
```

Comment:
It feels a little odd that this doesn't need *any* other information. But hey, if it works...

Reply:
We can add more, the options, the current `TokenResponse`, etc. I just though, lets do that if needed or requested.